alignment/src/main/java/org/jboss/pme/alignment/GAV.java

```diff
@@ -0,0 +1,44 @@
+package org.jboss.pme.alignment;
+
+import java.beans.Transient;
+
+public interface GAV {
```

Comment:
While I don't know if we will end up reusing code from PME, PME uses Atlas for GAV type objects e.g. https://github.com/Commonjava/atlas/blob/atlas-parent-0.17.1/identities/src/main/java/org/commonjava/maven/atlas/ident/ref/ProjectVersionRef.java
Perhaps we should do the same?

Reply:
Not sure yet but we'll look at it.

Reply:
I'll take a look at it and see if it makes sense to replace.